solr/docker/templates/Dockerfile.body.template

```diff
@@ -26,10 +26,11 @@
 #-#
 #-#
 
-# add symlink to /opt/solr, remove what we don't want
+# add symlink to /opt/solr, remove what we don't want.
+# Remove the Dockerfile because it might not represent the dockerfile that was used to generate the image.
 RUN set -ex; \
   (cd /opt; ln -s solr-*/ solr); \
-  rm -Rf /opt/solr/docs;
+  rm -Rf /opt/solr/docs /opt/solr/docker/Dockerfile;
```

Comment:
This will leave behind a docker/ folder with a README.md. Could just as well remove the folder?

Reply:
We can't because it also contains `docker/scripts` which is obviously very important 🙂 . We don't remove the other `README.md` files, so no real reason to do it explicitly here.